testing/agent-exporter/src/main/java/io/opentelemetry/javaagent/testing/exporter/OtlpInMemoryMetricExporter.java

```diff
@@ -34,7 +34,7 @@ void reset() {
 
   @Override
   public AggregationTemporality getAggregationTemporality(InstrumentType instrumentType) {
-    return AggregationTemporality.CUMULATIVE;
+    return AggregationTemporality.DELTA;
```

Comment:
@jack-berg this seems to make a micrometer test a bit flaky https://ge.opentelemetry.io/s/mwt7qlr5nwqig/tests/:instrumentation:micrometer:micrometer-1.5:javaagent:testBaseTimeUnit/io.opentelemetry.javaagent.instrumentation.micrometer.v1_5.TimerSecondsTest/testTimerWithBaseUnitSeconds()?expanded-stacktrace=WyIwIl0&top-execution=1
To reproduce insert `Thread.sleep(1_000);` between `timer.record()` calls. Any ideas where the issue is?